<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Band Selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="labelRenderType">
     <property name="text">
      <string>&lt;Render Type&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="renderBandWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageSingleBand">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Band</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QGroupBox" name="gbSingleBandWavelength">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Wavelength</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSlider" name="sliderSingleBand">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="RasterBandComboBox" name="cbSingleBand">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageMultiBand">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="5" column="0" colspan="4">
        <widget class="QGroupBox" name="gbMultiBandWavelength">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="title">
          <string>Wavelength Combination</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="toolTip">
          <string>Blue band</string>
         </property>
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QSlider" name="sliderMultiBandBlue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSlider" name="sliderMultiBandRed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="toolTip">
          <string>Red band</string>
         </property>
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="toolTip">
          <string>Green band</string>
         </property>
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSlider" name="sliderMultiBandGreen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="RasterBandComboBox" name="cbMultiBandRed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Select the band for red color values</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="RasterBandComboBox" name="cbMultiBandGreen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Select the band for green color values</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="RasterBandComboBox" name="cbMultiBandBlue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Select the band for blue color values</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageUnknown">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Renderer is not supported</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RasterBandComboBox</class>
   <extends>QComboBox</extends>
   <header>qps.layerconfigwidgets.rasterbands</header>
   <slots>
    <signal>bandChanged(int)</signal>
    <slot>setBand(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qpsresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sliderMultiBandRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cbMultiBandRed</receiver>
   <slot>setBand(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMultiBandRed</sender>
   <signal>bandChanged(int)</signal>
   <receiver>sliderMultiBandRed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMultiBandGreen</sender>
   <signal>bandChanged(int)</signal>
   <receiver>sliderMultiBandGreen</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMultiBandBlue</sender>
   <signal>bandChanged(int)</signal>
   <receiver>sliderMultiBandBlue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMultiBandGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cbMultiBandGreen</receiver>
   <slot>setBand(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMultiBandBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cbMultiBandBlue</receiver>
   <slot>setBand(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSingleBand</sender>
   <signal>bandChanged(int)</signal>
   <receiver>sliderSingleBand</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderSingleBand</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cbSingleBand</receiver>
   <slot>setBand(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
